PRINCIPLES.md

```diff
@@ -1,259 +1,62 @@
-</div>
+    If the actual and desired states differ for any reason, automated actions to reconcile them are initiated.
 
-<!--
-- If the software agents fail to bring the system's state in line with its desired state, a human operator is notified.
-- Software agents continuously check that the running system under management matches the desired state configuration, and if it does not, immediately either take remedial action to bring the system back in line with stated expectations or, if this cannot be done, alert a human operator that the system is no longer meeting expectations
-- Automated delivery: Delivery of the declarative descriptions, from the repository to runtime environment, is fully automated.
-- Software Agents: Reconcilers maintain system state and apply the resources described in the declarative configuration.
-- Closed loop: Actions are performed on divergence between the version controlled declarative configuration and the actual state of the target system.
-- This automation should serve its human users.
-  To this point, reconciliation automation for GitOps should include a way for humans to temporarily take back the reins as needed.
-- Implies observability. State of the running system must be observable. Ideally in same format as the configuration so we can compare like to like. 
-- >I agree. I think of this as an observability requirement and tend to say that "the current state of a running system must be observable" to mean the same thing. This is, in my opinion, required for state reconciliation. I didn't add a note there explicitly though, as that section was unfinished.
--->
+4. **The principle of operations through declaration**
 
+    The only mechanism through which a system can be intentionally operated on is through these principles.
 
-### 4. Operations through declaration
+## Notes
 
-<div style="background: #E3F2FD; border-left: 3px solid #0D47A1; padding:10px; margin-bottom:5px; font-style:italic">
-The mechanism through which change is applied to a system by either a human operator or another system is through the creation of a new declarative version of the desired state in the state store, not through directly interacting with the running system.
-</div>
-<!--
-- All normal operations should occur via the creation of a new uniquely named version, not through direct interaction ith the system under management
-- Break glass - exceptions are acceptable and quite likely. and credentials to access and mutate the system must still be available in almost all cases.
--->
+### Principle 3 Notes
 
-<!--
-## General notes on the GitOps principles
+- These differences could be due to the actual state drifting from the desired state, or the desired state changing intentionally.
+- The source of drift doesn't matter. Contrary to CIops, _any_ drift will trigger a reconciliation
+- By "continuous" we adopt the industry standard term to mean reconcilation continues to happen, not that it must be instantaneous
```

Comment:
```suggestion
```

Reply:
From the meeting today I will move the continuous point into the Glossary.

And it sounds like we should wait until after the first pre-release to further discuss moving the rest of these notes somewhere else like a best practices doc. I do like that idea in general.

Reply:
@moshloop I moved this in 509ebeb3e561ba1bd0117c96ba80d431a6b04da4 ✅